im3components/components.py

```diff
@@ -0,0 +1,34 @@
+from dataclasses import dataclass, field
+
+from im3components.gcam_cerf_expansion_plan import gcam_cerf_expansion
```

Comment:
i don't think this will be an issue unless/until we have tons of coupling, but this pattern will cause all the imports for every coupler to load when importing the registry, which could eventually cause dependency conflicts or startup lag. not really a problem at the moment though.

Reply:
Great point, I'll setup a solution for this.